### DISCUSSION TITLE
Holocene Calendar

### DISCUSSION THREAD

**Discussion by GPaso** [endless-sky/endless-sky#9476] [Ideas]
Make use of the Holocene calendar (adding 10'000 years to the calendar in use).

Or an alternative is use a new calendar starting with the First Human settlement on Mars as the year 1.

This can be explained in the Lore with a Calendar reform that took place internationally or intergalactically in order to focus more the attention on the Human Era or on the Space Era despite of which calendar any human on any planet were using.

**Comment by mOctave** [comment]
I think that the current calendar is good as it is, because it is much more relatable than either of your options.
The first human settlement on Mars thing sounds like technobabble for the sake of technobabble, plus people would wonder why the date was suddenly a thousand years in the past (from 2023).
I don't think I'd ever even _heard_ of the Holocene Calendar before you brought it up. That's a lot of digits to keep track of, and isn't very relatable.

For an overhaul plugin, either of these could be a great addition (including your lore tidbit). For the base game, they just don't make sense.

**Comment by Ferociousfeind** [comment]
The Holocene calendar is very fun, it's just putting a 1 in front of the year number we have now, but it also acknowledges that human societies have been around for ~10,000 years before christianity came along. It'd be 13013 instead of 3013

**Comment by LordInsane** [comment]
By precedent for these sort of games, simply having an unfamiliar year label directly in the interface seems to ward off practically all confusion about why the year has a lower number than the present (that NC made 1177 pass muster easily). That's just an argument against that counter-argument, though, not an argument for changing from ES' current calendar (it might be an argument for making if there are year labels and what it is if so moddable, though).
